deployments.json

```diff
@@ -1138,5 +1138,10 @@
     "name": "X Layer Mainnet",
     "chainId": 196,
     "url": "https://www.okx.com/web3/explorer/xlayer/address/0xcA11bde05977b3631167028862bE2a173976CA11"
+  },
+  {
+    "name": "WeaveVM Testnet",
+    "chainId": 9496,
+    "url": "https://explorer.wvm.dev/address/0x78A2377ED3d70779B85E1BB686eA1a723A7baD95"
```

Comment:
Is WeaveVM EVM compatible? I typically only merge non-standard addresses when a chain cannot support deployment to the standard address of `0xcA11bde05977b3631167028862bE2a173976CA11` (e.g. when it's a zk chain that computes addresses differently)